### DISCUSSION TITLE
RuntimeError: Server with name XXXX is not defined

### DISCUSSION THREAD

**Discussion by mckenna654** [Taxel/PlexTraktSync#1614] [Support]
EDIT

**Comment by glensc** [comment]
you have posted your plex token, plex url and same for trakt. anyone copying those values can access your data and accounts.

**Comment by glensc** [comment]
also, crashes need to be reported as issues:
- https://github.com/Taxel/PlexTraktSync/issues/new/choose

**Comment by glensc** [comment]
Seems the problem is in the only thing you obfuscated:  the server name. I took your servers.yml you posted here, and got full access to your server and data. you have shared your token and host addresses in unmodified form. also seems you have leaked here your trakt secrets. and, crashes need to be reported as issues, not discussions. discussions are between users, bugreports need to be created as issues to reach out to developers.

**Comment by glensc** [comment]
user disappeared silently, so no longer relevant

**Comment by glensc** [comment]
editing post content will not delete the leaked tokens. it can be still retrieved from edit history
